File: sample-code/fibonacci.pas
{ Pascal sample }
program Fibonacci;

function fib(n: Integer): Integer;
begin
  if n < 2 then
    fib := n
  else
    fib := fib(n-1) + fib(n-2);
end;

begin
  writeln('fib(10)=', fib(10));
end.

File: sample-code/fibonacci.dart
// Fibonacci (Dart)

int fib(int n) {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
}

void main() {
  for (var i = 0; i < 12; i++) {
    print('$i ${fib(i)}');
  }
}
